deps.edn

```diff
@@ -86,10 +86,11 @@
               "-Dclojure.spec.check-asserts=true"]}
 
   :test
-  {:extra-paths ["dev" "test"]
+  {:extra-paths ["test"]
    :extra-deps
    {lambdaisland/kaocha {:mvn/version "1.0.887"}
-    lambdaisland/kaocha-junit-xml {:mvn/version "0.0.76"}}}
+    lambdaisland/kaocha-junit-xml {:mvn/version "0.0.76"}
+    nrepl/nrepl {:mvn/version "0.9.0-beta4"}}}
```

Comment:
This was required because the dev profile apparently gets one for free, but after removing dev from test extra-paths, this is now explicit. Since one is requested in prod profile as well, wondering if we shouldn't just pull this up top level.